dts/bindings/spi/ite,it8xxx2-spi.yaml

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2022 ITE Corporation. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+description: ITE it8xxx2 SSPI
+
+compatible: "ite,it8xxx2-spi"
+
+include: spi-controller.yaml
+
+properties:
+  reg:
+    required: true
+
+  interrupts:
+    required: true
+
+  sckfreq:
+    type: int
+    required: true
+    description:
+      Clock frequency the SPI peripheral is being driven at, in MHz
+      0   SSPI_SCKFREQ_24MHZ
+      1   SSPI_SCKFREQ_12MHZ
+      2   SSPI_SCKFREQ_8MHZ
+      3   SSPI_SCKFREQ_6MHZ
+      5   SSPI_SCKFREQ_4MHZ
+      7   SSPI_SCKFREQ_3MHZ
+      Fill the SSPI_SCKFREQ_24MHZ in the sckfreq field,
+      it can be found in the project spi dts file.
+    enum:
+      - 0
+      - 1
+      - 2
+      - 3
+      - 5
+      - 7
+
+  spimode:
+    type: int
+    description:
+      SPI Mode
+      0  SSPI_MODE_0  Bit6 and Bit 5 are 00b
+      1  SSPI_MODE_1  Bit6 and Bit 5 are 01b
+      2  SSPI_MODE_2  Bit6 and Bit 5 are 10b
+      3  SSPI_MODE_3  Bit6 and Bit 5 are 11b
```

Comment:
what do these modes actually mean?

Reply:
Hi galak,

These constants represent the different clock phases which is defined as different modes and can be selected in the SPICTRL1 register. It behaves as follows:

Mode 0: SSCK is low in the idle mode. Data is sampled on the rising edge, and shifted on the falling edge.

Mode 1: SSCK is low in the idle mode. Data is sampled on the falling edge, and shifted on the rising edge.

Mode 2: SSCK is high in the idle mode. Data is sampled on the falling edge, and shifted on the rising edge.

Mode 3: SSCK is high in the idle mode. Data is sampled on the rising edge, and shifted on the falling edge.

Thanks for feedbacks & Best regards,
BJ

Reply:
Yes, please use `spi-cpol` and `spi-cpha` as your properties.  Your driver will process the properties to set the correct SSPI_MODE setting required by the ITE SPI controller.